src/main/java/com/hazelcast/jdbc/JdbcConnection.java

```
@@ -0,0 +1,384 @@
+    }
+
+    @Override
+    public void rollback(Savepoint savepoint) throws SQLException {
+        checkClosed();
+        throw JdbcUtils.unsupported("Rollback is not supported.");
+    }
+
+    @Override
+    public void releaseSavepoint(Savepoint savepoint) throws SQLException {
+        checkClosed();
+        throw JdbcUtils.unsupported("Savepoint is not supported.");
+    }
+
+    @Override
+    public Statement createStatement(int resultSetType, int resultSetConcurrency, int resultSetHoldability) throws SQLException {
+        checkClosed();
+        return createStatement();
+    }
+
+    @Override
+    public PreparedStatement prepareStatement(String sql, int resultSetType, int resultSetConcurrency, int resultSetHoldability) throws SQLException {
+        checkClosed();
+        return prepareStatement(sql);
+    }
+
+    @Override
+    public CallableStatement prepareCall(String sql, int resultSetType, int resultSetConcurrency, int resultSetHoldability) throws SQLException {
+        checkClosed();
+        throw JdbcUtils.unsupported("CallableStatement not supported");
+    }
+
+    @Override
+    public PreparedStatement prepareStatement(String sql, int autoGeneratedKeys) throws SQLException {
+        checkClosed();
+        throw unsupportedAutoGeneratedKeys();
+    }
+
+    @Override
+    public PreparedStatement prepareStatement(String sql, int[] columnIndexes) throws SQLException {
+        checkClosed();
+        throw unsupportedAutoGeneratedKeys();
+    }
+
+    @Override
+    public PreparedStatement prepareStatement(String sql, String[] columnNames) throws SQLException {
+        checkClosed();
+        throw unsupportedAutoGeneratedKeys();
+    }
+
+    @Override
+    public Clob createClob() throws SQLException {
+        checkClosed();
+        throw JdbcUtils.unsupported("Clob is not supported.");
+    }
+
+    @Override
+    public Blob createBlob() throws SQLException {
+        checkClosed();
+        return null;
```

`createBlob` should throw an unsupported exception